docs/src/components/productCore/CoreHero.tsx

```diff
@@ -35,8 +35,9 @@ export default function CoreHero() {
         >
           <IconImage width={28} height={28} name="product-core" /> MUI Core
         </Typography>
-        <Typography variant="h2" sx={{ maxWidth: 700, textAlign: 'center' }} gutterBottom>
+        <Typography variant="h2" sx={{ textAlign: 'center' }} gutterBottom>
           Ready to use components,
+          <br />
```

Comment:
There is a missing space in https://mui.com/core/, but new line feels even better.

Reply:
Much better, good catch!